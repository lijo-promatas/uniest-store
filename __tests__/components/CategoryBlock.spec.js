import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import CategoryBlock from '../../src/components/CategoryBlock';

it('renders <CategoryBlock /> component correctly', () => {
  const tree = renderer.create(<CategoryBlock
    totalPrice="100%"
    btnText="test"
    onBtnPress={() => {}}
    isBtnDisabled={false}
  />).toJSON();
  expect(tree).toMatchSnapshot();
});
