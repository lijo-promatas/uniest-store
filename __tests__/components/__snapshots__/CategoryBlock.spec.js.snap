// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <CategoryBlock /> component correctly 1`] = `null`;
